xchange-acx/src/main/java/org/knowm/xchange/acx/AcxApi.java

```diff
@@ -59,6 +54,52 @@ AcxOrderBook getOrderBook(
   @Path("/trades.json?market={market}")
   List<AcxTrade> getTrades(@PathParam("market") String market) throws IOException;
 
+  /**
+   * Get recent trades on market, each trade is included only once. Trades are sorted in reverse
+   * creation order.
+   *
+   * @param market Unique market id. It's always in the form of xxxyyy, where xxx is the base
+   *     currency code, yyy is the quote currency code, e.g. 'btcaud'. All available markets c an be
+   *     found at /api/v2/markets.
+   */
+  @GET
+  @Path(
+      "/trades/my.json?market={market}&access_key={access_key}&tonce={tonce}&signature={signature}&limit={limit}&order={order}")
+  List<AcxTrade> getMyTrades(
+      @PathParam("access_key") String accessKey,
```

Comment:
use one method with @QueryParam instead of @PathParam
you can pass null if you do not want it in the request so one method will be enough

Reply:
This wasn't working initially, turned out it required a small change to the digest code to make it work with a single method.